sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/ServiceItemLease.java

```diff
@@ -20,6 +22,7 @@
 /**
  * Document service lease.
  */
+@JsonSerialize(using = ServiceItemLease.ServiceItemLeaseJsonSerializer.class)
```

Comment:
Please add unit test scoped to serialization.

Reply:
Test added.